src/main/resources/avro/sequenceAnnotations.avdl

```diff
@@ -0,0 +1,227 @@
+  Identifier from a public database
+  */
+  record ExternalIdentifier {
+    /**
+    The source of the identifier.
+    (e.g. `Ensembl`)
+    */
+    string database;
+
+    /**
+    The ID defined by the external database.
+    (e.g. `ENST00000000000`)
+    */
+    string identifier;
+
+    /**
+    The version of the object or the database
+    (e.g. `78`)
+    */
+    string version;
+  }
+
+  /**
+  Type defining a collection of attributes associated with various protocol
+  records.  Each attribute is a name that maps to an array of one or more
+  values.  Values can be strings, external identifiers, or ontology terms.
+  Values should be split into the array elements instead of using a separator
+  syntax that needs to parsed.
+  */
+  // TODO: how are multiple instances of a given attribute vs multiple values
+  // for an attribute distinguished
+  record Attributes {
+    map<array<union {string, ExternalIdentifier, OntologyTerm}>> vals = {};
+  }
+  
+  /**
+  Continuous numerical annotation along a path.
+  */
+  record Wiggle {
+    /**
+    The path for the region being annotated.
+    Note that we are fusing parts of UCSC BedGraph and Wiggle syntax.
+    The region is being fully annotated, but can be divided into bins.
+    If you have gaps, you need to define a sequence of such Wiggles.
+    */
+    /* TODO: path not used yet, as graph code is not working in server.
+       Fields below are copied directly from variants.avdl for now.
+    Path path; */
+
+    /**
+    The reference on which this feature occurs.
+    (e.g. `chr20` or `X`)
+
+    If the API server supports the "classic" mode, this field must not be null.
+    */
+    union { null, string } referenceName = null;
+
+    /**
+    The start position at which this feature occurs (0-based).
+    This corresponds to the first base of the string of reference bases.
+    Genomic positions are non-negative integers less than reference length.
+    Features spanning the join of circular genomes are represented as
+    two features one on each side of the join (position 0).
+
+    If the API server supports the "classic" mode, this field must not be null.
+    */
+    union { null, long } start = null;
+
+    /**
+    The end position (exclusive), resulting in [start, end) closed-open interval.
+    This is typically calculated by `start + referenceBases.length`.
+
+    If the API server supports the "classic" mode, this field must not be null.
+    */
+    union { null, long } end = null;
+
+    /**
+    The values associated to this region.
+    If this list contains _count_ elements, then the region is divided
+    as cleanly as possible into _count_ bins of equal width.
+    We thus define a numerical function:
+
+    value(position):
+      if position < start or position >= start + length:
+        return None
+      else:
+        return array[floor((position - start)*count/length)]
+    */
+    array<float> values = [];
+  }
+
+  /*
+  A set of wiggle annotations
```

Comment:
There isn't currently an implementation for Wiggle.  I'm thinking we either write one or remove it from the schema for the time being.  I'm leaning towards removing it for the time being.